meetings/azure.yaml

```diff
@@ -0,0 +1,17 @@
+project: Azure working group
+meeting_id: Azure
+project_url: https://github.com/ansible/community/wiki/azure
+agenda_url: https://github.com/ansible/community/issues?q=is:open+label:meeting_agenda+label:azure
+chair: Matt Davis (nitzmahone), Kylie Liang (kyliel)
+description: |
+
+  We discuss everything related to Azure every week on Wednesday.
+
+  If you have something to discuss (e.g. a PR that needs help), add your
+  request to the meeting agenda and join the IRC channel.
+schedule:
+- time: '0000'
+  day: Wednesday
```

Comment:
Isn't it technically Thursday at midnight (the morning of Thursday). These meetings are at 10am on a Thursday for me and I'm UTC+10. The ics would have to be changed.